check_mk-agent.install

```diff
@@ -1,7 +1,8 @@
 post_install() {
-	systemctl restart xinetd.service
+    systemctl daemon-reload
+    systemctl enable --now check_mk-agent.socket    
 }
 post_upgrade() {
```

Comment:
```suggestion
post_upgrade() {
  # if a upgrade from xinetd restart xinitd
  if [ $(vercmp $2 1.6.0-1) -lt 0 ]; then
      systemctl restart xinetd.service
  fi
  systemctl daemon-reload
```

Reply:
would be nice so a upgrade is working and the socket is starting without get blocked by xinitd :6556